### DISCUSSION TITLE
Can't setup Twitter on Hubs Cloud

### DISCUSSION THREAD

**Discussion by smakhtin** [mozilla/hubs#5122] [Hubs Cloud Questions] [answered]
Hello, I'm trying to setup Twitter integration on Hubs cloud, but looks like I'm messing something with API keys, because server always responds with 400 error ("Upload Failed") when i trying to post tweet,

As i understood, Twitted changed theirs API keys UI, but Hubs documentation wasn't updated.

Here is how it's looks right now:
<img width="648" alt="image" src="https://user-images.githubusercontent.com/271810/153637131-53852616-66a0-427e-ad54-e82126a2e115.png">

@robinkwilson can you please help to figure out how to link these keys with Hubs Cloud Admin input fields?

**Comment by robinkwilson** [comment, marked as answer]
From the [Hubs Docs: Twitter integration](https://hubs.mozilla.com/docs/hubs-cloud-enable-media-browser.html#twitter): 

`You'll need Consumer keys (API Key and Secret) and Authentication Tokens (Access Tokens and Secret)`.

Below Hubs names -> Twitter names:

Twitter Access Token -> Access Token 
Twitter Access Token Secret -> Access Token Secret
Twitter Consumer Secret -> Consumer Keys API Key Secret
Twitter Consumer Key -> Consumer Keys API Key

I've made mistakes with the mapping too a few times because it's not in order in the Hubs Admin Panel. Can you regenerate the Twitter keys (if you can) and paste them in to the Admin Panel again?


<img width="578" alt="image" src="https://user-images.githubusercontent.com/29560735/153692027-5bf5939a-a962-4121-9fd4-212e13a9589b.png">
